aptos-move/framework/aptos-stdlib/sources/any.spec.move

```diff
@@ -15,10 +15,12 @@ spec aptos_std::any {
         ensures [abstract] from_bcs::deserializable<T>(result.data);
     }
 
-    spec unpack<T>(x: Any): T {
+    spec unpack<T>(self: Any): T {
         use aptos_std::from_bcs;
-        include UnpackAbortsIf<T>;
-        ensures result == from_bcs::deserialize<T>(x.data);
+        include UnpackAbortsIf<T> {
+            x: self
```

Comment:
Perhaps also rename in the Schema to avoid this.

Reply:
I don't rename schemas because I am not sure whether external developers might use them and changing the name may break their specs though they can simply update them. What do you think?

Reply:
Good point.

I guess though for the occasions we are looking at here this would be unlikely, and also the previous `x` was not a good API. So lets risk breaking them.

Reply:
done